docs/man/docker-inspect.1.md

```diff
@@ -158,8 +158,13 @@ To get information on a container use its ID or instance name:
             "Running": true,
             "StartedAt": "2014-04-04T21:33:54.16259207Z",
         },
-        "Volumes": {},
-        "VolumesRW": {},
+        "MountPoints": {
+          "/data": {
+            "Source": "/data",
+            "Destination": "/data",
+            "RW" false
```

Comment:
Question on this one :
- Seems like there is a type (missing a ``:``)
- the ``:ro`` option is mapped to ``"RW": false`` and ``:rw`` to ``"RW": true``, right ? Then what about the ``z`` option, and the composition of them ``rw,z`` ? (for reference : https://github.com/docker/docker/pull/12572 )